addons/l10n_fr_pdp_reports/__manifest__.py

```diff
@@ -13,13 +13,16 @@
         "account_peppol",
         "account_reports",
         "l10n_fr",
+        "l10n_fr_pdp",
     ],
     "data": [
         "security/ir.model.access.csv",
         "data/pdp_cron.xml",
+        "data/flux10_transaction_report_template.xml",
+        "views/res_company_views.xml",
+        "views/account_move_views.xml",
     ],
-    "installable": True,
-    "auto_install": False,
-    "application": False,
+    'installable': True,
+    'auto_install': True,
     "license": "LGPL-3",
 }
```

Comment:
i suppose it's a mistake but we have a lot of change for l10n_fr_pdf_reports in the commit of the file format ?